### ISSUE TITLE
Wrong fio command breaks disk speed tests

### ISSUE THREAD

**Issue by StevenNL2000** [masonr/yet-another-bench-script#27]
I tried to run YABS on a system that I do not manage, and got the following output for the disk speed tests:
```plaintext
fio Disk Speed Tests (Mixed R/W 50/50):
---------------------------------
Block Size | 4k            (IOPS) | 64k           (IOPS)
  ------   | ---            ----  | ----           ---- 
Read       |                   () |                   ()
Write      |                   () |                   ()
Total      | 0.00 KB/s        (0) | 0.00 KB/s        (0)
           |                      |                     
Block Size | 512k          (IOPS) | 1m            (IOPS)
  ------   | ---            ----  | ----           ---- 
Read       |                   () |                   ()
Write      |                   () |                   ()
Total      | 0.00 KB/s        (0) | 0.00 KB/s        (0)
```

The culprit turns out to be a Python package called Fiona, which also has a `fio` command that has nothing to do with the one expected by the script:
https://github.com/Toblerity/Fiona/blob/master/docs/cli.rst

For this particular case, I think it would be useful to have a flag in YABS to force external binaries to be used even when local ones are detected.

**Comment by masonr** [comment]
Thanks for the bug report!

> I think it would be useful to have a flag in YABS to force external binaries to be used even when local ones are detected.

Sounds like a good idea, I can add that in.

**Comment by admdly** [comment]
I've noticed that on some machines with low memory and an older version of fio (3.17) as supplied by the OS repos causes a fault which prevents the job from running.

I'd suggest, for repeatability and to avoid such issues, that YABS defaults to the provided binaries and only uses the OS binaries if requested.

**Comment by Theo-rhone** [comment]
In OSX, the problem is due to :
- The image libaio is not supported

Some type the problem is due to the shared memory segment.

**Comment by masonr** [comment]
Fixed in 624e7e9

Added the `-b` option to allow users to specify that they wish to use the pre-compiled binaries in the repo rather than any locally installed packages.
